project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_quickpost.scss

```diff
@@ -1,3 +1,10 @@
+.quickpost-title {
+
+  &:focus::-webkit-input-placeholder {
```

Comment:
@guytepper 
this won't work in IE9

we'll probably have to remove the placeholder on focus, you can probably use:
```js
ng-placeholder=if <var> then <placeholder>
ng-focus=set <var> true
ng-blur=set <var> false
```